shared/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("UNUSED_VARIABLE")
```

Comment:
Why do we need this?

Reply:
Our declared sourcesets  (commonMain, iosMain, androidMain, etc.) all appear as unused, giving a warning in the editor. Adding this suppressor remove those warnings, for a clean state when reading that file 🙂